### DISCUSSION TITLE
Portfolio Analysis

### DISCUSSION THREAD

**Discussion by aisyah-zakiah** [NREL/REopt-API-Analysis#159] [1 - General REopt Questions]
Hi!

I tried using the Portfolio Analysis feature in REopt web, but it failed to run. I have filled all required fields and left others using default values.
Any suggestions on what to do?

**Comment by adfarth** [comment]
Hello, I was just able to run a quick test Portfolio Analysis. Could you please share exactly which input fields you were utilizing?

**Reply by aisyah-zakiah** [reply]
Hi @adfarth ,

I am sharing below how I filled the input fields. After I click 'Get Result,' nothing happens, and the portfolio analysis is not saved in the saved evaluations too.

![image](https://github.com/NREL/REopt-Analysis-Scripts/assets/173452456/60664252-cf98-4504-9db2-1cc759b8603f)
![image](https://github.com/NREL/REopt-Analysis-Scripts/assets/173452456/e2aafae5-8fc0-4c0d-b3c1-6ae0e23b7b8f)
![image](https://github.com/NREL/REopt-Analysis-Scripts/assets/173452456/ec56edbf-349e-4827-bd10-9f4fe42c0e99)
![image](https://github.com/NREL/REopt-Analysis-Scripts/assets/173452456/23d0eb36-8c25-49a2-a95b-17f672ce5f78)
![image](https://github.com/NREL/REopt-Analysis-Scripts/assets/173452456/23c5db99-59c9-401b-b916-f51988925cc9)

**Reply by adfarth** [reply]
Thank you for sharing this! Our team has fixed the issue causing this error and you should now be able to run this analysis.
